scripts/caclmgrd

```diff
@@ -347,21 +350,23 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
         return allow_internal_docker_ip_cmds
 
 
-    def generate_fwd_traffic_from_host_to_soc(self, namespace):
+    def generate_fwd_traffic_from_host_to_soc(self, namespace, new_config_db_connector=None):
 
         fwd_dualtor_grpc_traffic_from_host_to_soc_cmds = []
+        if not new_config_db_connector:
```

Comment:
This logic make other difficult to understand the code behavior.
Suggest always use the connector from parameter.

Reply:
Good point, update code according to your suggestion.